src/js/electron/tron/windowManager.ts

```diff
@@ -207,6 +207,12 @@ export default function windowManager(
       }
     },
 
+    openHidden() {
+      // only open hidden window if one doesn't already exist
+      if (this.getWindows().find((w) => w.name === "hidden")) return
+      this.openWindow("hidden")
```

Comment:
Let's make methods called

`getAll()`

`getVisible()`

`getHidden()`

Then we could use them in all the places that make sense.